packages/writing-style/vale-styles/commercetools/Acronyms.yml

```diff
@@ -24,11 +24,13 @@ exceptions:
   - JAR
   - JSON
   - JSX
+  - MDX
```

Comment:
This exposes a bit of a conflict when using the writing style checker on the kit itself.  "MDX" should not be used on commercetools platform documentation without explaining it, so I believe it does not belong into this whitelist.  

I did not find out yet how to add or override the list for a given repository or website, that would be the better approach mid-term (even inside the parts of the commercetools documentation - e.g. in the Merchant Center documentation you shouldn't use "JSX" without explaining it but in the custom apps documentation that's perfectly fine.  

If it's possible to override the rule selection or severity locally we could duplicate the Acronyms rule into topic-specific ones, e.g.  for Javascript related topics, one for User Documentation and one for general API documentation.   

No need to escalate it over MDX specifically though.

Reply:
Hmm I see...do you want to open an issue to track this?